azext_iot/common/utility.py

```diff
@@ -351,6 +355,36 @@ def unpack_msrest_error(e):
     return op_err
 
 
+def handle_service_exception(e):
+    from azure.cli.core.azclierror import (
```

Comment:
Would be nice to include a link to the error guidance documentation, especially if the azure cli team decides to change the types.